README.md

```diff
@@ -146,7 +146,37 @@ data directory.
 `tezos-{accuser, endorser}` have configurable node address, so that they can be used with both
 remote and local node.
 
-### Systemd units on other systems
+### Launchd services on macOS
+
+`tezos-accuser-<proto>`, `tezos-baker-<proto>`, `tezos-endorser-<proto>` formulas
+provide backround services for running corresponding daemons.
+
+Since `tezos-node` and `tezos-signer` provide multiple services they are provided
+in a dedicated meta-formulas. These formulas doesn't install any binaries and only provide
+backround services.
```

Comment:
```suggestion
Since `tezos-node` and `tezos-signer` need multiple services they are provided
in dedicated meta-formulas. These formulas don't install any binaries and only add
background services.
```
A couple of typos and I tried to not repeat "provide" too much.